subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c

```diff
@@ -66,6 +68,7 @@ static void isr_tx(void *param);
 static void isr_rx(void *param);
 static void isr_done(void *param);
 static void isr_abort(void *param);
+static void isr_abort_too_late(void *param);
```

Comment:
```suggestion
static void isr_late(void *param);
```

Short `isr_late` should suffice.